### ISSUE TITLE
Infinite Y Direction

### ISSUE THREAD

**Issue by flippygat** [AlexSTV/Voxelmetric#62]
Would be great if the player could travel as far down as they want.

**Comment by AlexSTV** [comment]
Agreed, this would be nice. This is part of the reason why in alpha 3 I changed terrain generation to run per chunk, not per chunk column. So that if you want you can create new chunks and generate terrain for them outside a predefined y min and max height. This should be the way things work eventually.
